src/CentralPackageVersions/Sdk/Sdk.targets

```diff
@@ -28,6 +28,8 @@
     <EnableCentralPackageVersions
       Condition="'$(EnableCentralPackageVersions)' == '' And (!Exists('$(CentralPackagesFile)') Or Exists('$(MSBuildProjectDirectory)\packages.config') Or '$(MSBuildProjectExtension)' == '.vcxproj' Or '$(MSBuildProjectExtension)' == '.ccproj' Or '$(MSBuildProjectExtension)' == '.nuproj')">false</EnableCentralPackageVersions>
 
+    <EnablePackageVersionOverrideWithoutCentralVersion Condition="'$(EnablePackageVersionOverrideWithoutCentralVersion)' == '' And '$(EnableCentralPackageVersions)' != 'false'">true</EnablePackageVersionOverrideWithoutCentralVersion>
```

Comment:
There's no need to default this property to any value as we'll only look for the user to turn it off by setting it to `false`.  Let's remove this.

Reply:
Yeah I wasn't sure whether checking for false and setting true by default was the right choice. I removed this, thanks for the guidance here.